src/connection-manager/manifests.ts

```diff
@@ -244,16 +257,24 @@ async function updateRevocations(): Promise<void> {
 }
 
 /**
- * Deletes validators that are older than an hour.
+ * Deletes validators that are older than 30 days.
+ * UNL validators are not deleted even if they are older than 30 days.
  *
  * @returns Void.
  */
-async function purgeOldValidators(): Promise<void> {
-  const oneWeekAgo = new Date()
-  oneWeekAgo.setDate(oneWeekAgo.getDate() - 7)
+export async function purgeOldValidators(): Promise<void> {
+  const thirtyDaysAgo = new Date()
```

Comment:
Do we know how much more disk-space is required due to the changes in the PR? Does this PR need to be accompanied by such infra updates?